### PR TITLE
Improve AnimatedSprite2D description in the class reference

### PR DESCRIPTION
**Pull request by Calinou**
This supersedes https://github.com/godotengine/godot/pull/38601.

### CODE REVIEW

File: doc/classes/AnimatedSprite2D.xml [hunk 1]
```diff
@@ -28,8 +28,8 @@
 		</method>
 	</methods>
 	<members>
-		<member name="animation" type="StringName" setter="set_animation" getter="get_animation" default="&amp;&quot;default&quot;">
-			The current animation from the [code]frames[/code] resource. If this value changes, the [code]frame[/code] counter is reset.
+		<member name="animation" type="StringName" setter="set_animation" getter="get_animation" default="@&quot;default&quot;">
```
Comment (akien-mga) [inline doc/classes/AnimatedSprite2D.xml:31]: Won't doctool revert that change?

Comment (akien-mga) [reply doc/classes/AnimatedSprite2D.xml:31]: Yes: https://github.com/godotengine/godot/runs/6413040535?check_suite_focus=true

Comment (Calinou) [reply doc/classes/AnimatedSprite2D.xml]: Fixed, it was a rebasing mistake.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**